src/App/App.js

```diff
@@ -1,9 +1,10 @@
 import React, { Component } from 'react';
 import { BrowserRouter, Route, Link } from 'react-router-dom';
 import Login from '../Login/Login';
+import ShowPage from '../ShowPage/ShowPage'
 import Header from '../Header/Header';
 import MovieContainer from '../MovieContainer/MovieContainer';
-import { getMovies } from '../apiCalls';
+import { getMovies, getSingleMovie } from '../apiCalls';
```

Comment:
I don't think we need `getSingleMovie` to be imported in `App` since we are now invoking our fetch request in a different file.